roles/infrastructure/defaults/main.yml

```diff
@@ -134,6 +134,10 @@ infra__gcp_project:                 "{{ common__gcp_project }}"
 infra__gcp_storage_location_data:   "{{ infra.gcp.storage.path.data | default([infra__storage_name, infra__data_path] | join('-')) }}"
 infra__gcp_storage_location_logs:   "{{ infra.gcp.storage.path.logs | default([infra__storage_name, infra__logs_path] | join('-')) }}"
 
+infra__gcp_cloud_router_name:       "{{ infra.gcp.network.router.name | default([infra__namespace, 'router'] | join('-')) }}"
```

Comment:
These should be declared as standard suffixes and then referenced here, not input as static strings.

Reply:
Thanks @Chaffelson. Updated the code with those changes in commit [886b15e](https://github.com/cloudera-labs/cloudera.exe/pull/115/commits/886b15e283d45258f4779aa104e124116d0da5ca).